styles.css

```diff
@@ -1,3 +1,230 @@
+html,
 body {
-  background-color: white;
+  scroll-behavior: smooth;
+  margin: 0;
+  padding: 0;
+  font-family: Helvetica, sans-serif;
+}
+
+/* Navbar */
+nav {
+  background-color: black;
+  min-height: 8vh;
+}
+
+ul {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  padding: 3vh 3vw;
+}
+
+li:hover {
+  background-color: rgba(200, 200, 200, 0.5);
+  font-weight: bold;
+}
+
+nav a {
+  text-decoration: none;
+  font-size: 1.5vw;
+  color: rgba(255, 255, 255, 0.8);
+}
+
+nav a:hover {
+  color: gold;
+  text-decoration: underline;
+}
+
+/* Main content */
+
+/* Generic shared CSS */
+#aboutme,
+#portfolio {
+  min-height: 100vh;
+  min-width: 98vw;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(600px, 1fr));
+  grid-template-rows: auto;
+}
+
+#aboutme,
+#contact {
+  background-color: rgba(200, 200, 200, 0.5);
+}
+
+/* Intro homepage */
+.home {
+  min-height: 91vh;
+  min-width: 98vw;
+}
+#home-image {
+  padding: 2vw;
+}
+
+#home-image > img {
+  height: 80vh;
```

Comment:
this line gave the picture some weird cropping on smaller sizes

<img width="1394" alt="image" src="https://user-images.githubusercontent.com/40119924/189635170-99892274-5c00-49f3-a0ae-2ec052229c5f.png">


prob better to do away with it and to size the container instead to make the picture smaller